server/datastore/mysql/software_installers.go

```diff
@@ -287,3 +323,98 @@ WHERE
 
 	return &dest, nil
 }
+
+func tmplNamedSQLCaseHostSoftwareInstallStatus(alias string) string {
```

Comment:
:D Looks like we'll have a couple of those come integration time!

Reply:
IKR! I thought this might be an interesting way to skin it to just return the format string. The idea was to allow the caller to make the call to `sql.Named` so that the caller could manage other named params too.

Reply:
Yeah I refactored my original query to have something like that too that I use elsewhere. That's fine, we can dedupe in the cleanup story!